.github/workflows/ci.yml

```diff
@@ -642,6 +642,41 @@ jobs:
           path: diagnostics*.gz
           retention-days: 1
 
+  benchmarks:
+    name: Benchmarks
+    needs: local_cluster_test
```

Comment:
since this is only benchmark code, it should not need `local_cluster_test` only primary build

Reply:
I thought if tests doesn't pass, theres no need to benchmark because theres an issue and benchmarking it wouldn't be useful?

Reply:
lets make depend on `check` this only excercise code  path